apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -32,8 +32,8 @@ type OpsRequestSpec struct {
 	// Specifies the name of the Cluster resource that this operation is targeting.
 	//
 	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="forbidden to update spec.clusterRef"
-	ClusterRef string `json:"clusterRef"`
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="forbidden to update spec.clusterName"
+	ClusterName string `json:"clusterName"`
```

Comment:
Incompatibility with the old API may cause existing clients (such as kbcli or cloud) to malfunction.